.konflux/scripts/subscription-manager/subscription-manager-bro.sh

```diff
@@ -84,31 +82,25 @@ function usage {
     echo "the actual activation key as a value."
     echo "   Find where to get the secret from ${SECRET_INFO_URL}"
```

Comment:
What's now missing is that we should pass `ACTIVATION_KEY` parameter to `buildah*` task. If we add it to step (1), I'm afraid it's easy for it to get lost. I suggest to restore step (2) but obviously change its content.

Reply:
Added in https://github.com/stackrox/stackrox/pull/12147/commits/f91a0d43cfb2d774d484a6c012182f93faa93100